javascript2.lexer/external/jflex-1.4.3-license.txt

```diff
@@ -0,0 +1,308 @@
+Name: JFlex
+Version: 1.4.3
+Description: The Fast Scanner Generator
+License: GPL
```

Comment:
The license verifier says: `contains a license body which does not match that in nbbuild/licenses/GPL`

The pretext here is relevant (-> it explicitly states, that the license relevant for generate code is the license of the source grammar, not that of JFlex). Please add the license also as GPL-flex to `nbbuild/licenses` and adjust the license reference here.